README.md

```diff
@@ -2,54 +2,125 @@
+
+```bash
+brew tap kube-compose/homebrew-kube-compose
 ```
-kube-compose -e mybuildid up
+
+Now you've added our custom tap, you can download with the following command:
+
+```bash
+brew install kube-compose
 ```
 
-The target namespace and service account token are loaded from the context set in `~/.kube/config`. This means that Openshift Origin Client Tools' `oc login` and `oc project` commands can be used to configure kube-compose's target namespace and service account.
+To upgrade your proxy to the latest stable release use the following command:
 
-If no `~/.kube/config` exists and kube-compose is run inside a pod in Kubernetes, the pod's namespace becomes the target namespace, and the service account used to create pods and services is the pod's service account.
+```bash
+brew upgrade kube-compose
+```
 
-The namespace can be overriden via the `--namespace` option, for example: `kube-compose --namespace ci up`.¯
+Otherwise download the binary from https://github.com/jbrekelmans/kube-compose/releases, and place it on your `PATH`.
 
-# Advanced usage
-If you require that an application is not started until one of its dependencies is healthy, you can add `condition: service_healthy` to the `depends_on`, and give the dependency a [Docker healthchecks](https://docs.docker.com/engine/reference/builder#healthcheck).
+## Getting Started
 
-Docker healthchecks are converted into [Readiness Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
+### Prerequisites
 
-# Building
-```
-go build -o kube-compose .
-```
-# Building (docker)
-```
-docker-compose build
-```
+NA
+
+### Testing
 
-# Testing
 Use `kubectl` or `oc` to set the target Kubernetes namespace and the service account of kube-compose.
 
 Run `kube-compose` with the test [docker-compose.yml](test/docker-compose.yml):
-```
+
+```bash
 (cd test && ../kube-compose --env-id test123 up)
 ```
+
 This writes to the directory `test/output` the created Kubernetes resources.
 
 To clean up after the test:
-```
+
+```bash
 kubectl delete $(kubectl get all -lenv=test123 -oname)
 ```
+
+### Build And Package
+
+You can compile the kube-compose binary using either Go or Docker-compose.
+
+Using Go:
+
+```go
+go build -o kube-compose .
+```
+
+Using Docker-compose
+
+```bash
+docker-compose build
+```
+
+## Commands
+
+The following is a list of all available commands:
+ 
+```bash
+Available Commands:
+  up          A brief description of your command
+  down        A brief description of your command
+  help        Help about any command
+```
+
+The `kube-compose up` command behaves in the same manner `docker-compose up` would be expected, however kube-compose loads the traditional docker-compose.yml file and translates the definition into kubernetes native manifests that allow the services to be created while respecting all docker-compose routing and functionality while utilizing kubernetes resources behind the scenes.
```

Comment:
This sounds a bit better:

> Intuitively, the kube-compose up mirrors functionality of docker-compose up, but runs containers on a Kubernetes cluster instead of on the host docker.